packit/upstream.py

```diff
@@ -1618,11 +1621,14 @@ def create(self, create_symlink: Optional[bool] = True) -> str:
         dir_name = f"{package_name}-{self.version}"
         logger.debug(f"Name + version = {dir_name}")
 
-        env = {
+        env = self.upstream.package_config.get_base_env()
+
+        env = env | {
             "PACKIT_PROJECT_VERSION": self.version,
             "PACKIT_PROJECT_NAME_VERSION": dir_name,
         }
         env = env | self.upstream.package_config.get_package_names_as_env()
+
```

Comment:
nitpick: the spacing additions and changes are all over the place. In some places you group the `env` modifications together, but like in here, you don't. At least be consistent or minimize the changes.